golang/internal/cue_validator/cue_validator_test.go

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
+package main
+
+import (
+	"testing"
+)
+
+func Test_target(t *testing.T) {
+	// Test an invalid policy.
+	invalid_policy := LoadResource(t, "invalid_policy.cue")
+	value := target(invalid_policy)
+	if value != nil {
+		t.Errorf("Expected errors but got none.")
+	}
+
+	// Test getting the target from a valid policy.
+	valid_policy := LoadResource(t, "valid_policy.cue")
+
+	value = target(valid_policy)
+	if value == nil {
+		t.Errorf("Unexpected error.")
+	}
+	// GoLang doesn’t provide any built-in support for assert.
+	expected_target := "urllib3/urllib3"
+	if GetGoString(value) != expected_target {
+		t.Errorf("%s does not match expected value %s.", GetGoString(value), expected_target)
+	}
+}
+
+func Test_validatePolicy(t *testing.T) {
+	// Load resources.
+	invalid_policy := LoadResource(t, "invalid_policy.cue")
+	valid_policy := LoadResource(t, "valid_policy.cue")
+	invalid_prov := LoadResource(t, "invalid_provenance.json")
+	valid_prov := LoadResource(t, "valid_provenance.json")
+	valid_prov2 := LoadResource(t, "valid_provenance2.json")
```

Comment:
Can we have a comment on the differences between `valid_prov` and `valid_prov2` ? Or I think we can rename the variables here.

Reply:
See the test names in [commit](https://github.com/oracle-samples/macaron/pull/90/commits/08e04a466b17dc863855f47054c5880d2d2b701c).